python-lib/publish/action.yml

```diff
@@ -18,8 +18,33 @@ runs:
     - uses: actions/setup-python@v2
       with:
         python-version: ${{ inputs.python-version }}
+    - name: Get feature branch name
+      if: ${{ startsWith(github.ref, 'refs/heads/feature/') }}
+      id: feature
+      run: |
+        ref="${{ github.ref }}"
+        echo "::set-output name=feature-name::${ref#refs/heads/feature/}"
+      shell: bash
+    - uses: LedgerHQ/actions/gemfury/latest-version
+      if: ${{ inputs.public == 'false' && startsWith(github.ref, 'refs/heads/feature/') && env.PYPI_FULL_ACCESS_TOKEN != '' }}
+      id: fury
+      with:
+        feature-name: ${{ steps.feature.outputs.feature-name }}
+      env:
+        FULL_ACCESS_TOKEN: ${{ env.PYPI_FULL_ACCESS_TOKEN }}
     - name: Set env vars
       run: |
+        publish_feature="false"
+        if [[ -n "${{ steps.fury.outputs.next-feature-tag }}" ]];
+          if [[ $(grep -E '__version__ = ".+"' setup.py) ]]; then
+            echo "Temporarily updating version with feature tag ${{ steps.fury.outputs.next-feature-tag }}."
+            sed -i 's/__version__ = "\(.\+\)"/__version__ = "\1${{ steps.fury.outputs.next-feature-tag }}"/' setup.py
+            publish_feature="true"
+          else
+            echo "Can't find __version__ to update, skipping feature publish."
```

Comment:
maybe we should abort ?
it's difficult to find way the pkg wasn't published in the logs :-)